miden-lib/src/notes/mod.rs

```diff
@@ -1,3 +1,5 @@
+use std::env;
```

Comment:
Is there a downside to using `#![cfg_attr(not(feature = "std"), no_std)]` here? I think that's what we do for `MidenLib`.

Reply:
Only that we might not be able to run `miden-base` in the browser then, right? Which might be relevant for the wallet?

Reply:
Don't we do the same thing for `midenlib` and Miden `stdlib` and they are still usable in `no_std`?

Reply:
I think `lib.rs` in `miden-lib` does not work with no_std? 

https://github.com/0xPolygonMiden/miden-base/blob/main/miden-lib/src/lib.rs#L1-L31

Reply:
hmm - I think it should work in `no_std`. My understanding is that `include_bytes!` macro works in `no_std` (e.g., based on [this](https://github.com/rust-lang/rust/issues/36272)).

Reply:
Let me try, I think, the problem is `env!` rather that `include_bytes!`